changelogs/fragments/1057-module_zos_mvs_raw_errors_with_long_multi_line_quoted_string_in_content_field.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+  zos_mvs_raw:
+  - module wasn't able to wrote properly a JCL with multi-line | or |1-9.
+    Change now validate the length of lines and allow multi-line declaration with | or |1-9.
```

Comment:
I think this description needs to mention that the actual bug truncated the first one or two characters writing an syntactically incorrect JCL and now two empty spaces are added into the content because some programs expect to have the first or first two columns as empty spaces.